examples/docs/public/admin/index.html

```diff
@@ -1,4 +1,4 @@
-<!DOCTYPE html>
+<!doctype html>
```

Comment:
I don't think this is necessary to lowercase `doctype` here, and many conventions have it in uppercase. Let's leave this uppercased to follow conventions here.

Reply:
Yeah, I don't love it either.

```suggestion
<!DOCTYPE html>
```